docs/dev/table/sql.md

```diff
@@ -497,6 +497,23 @@ FROM (
 {% endhighlight %}
       </td>
     </tr>
+
+    <tr>
+      <td>
+        <strong>In</strong><br>
+        <span class="label label-primary">Batch</span>
+      </td>
+      <td>
+      Returns true if an expression exists in a given table sub-query. The sub-query table must consist of one column. This column must have the same data type as the expression.
```

Comment:
What do you mean with "enhanced"? We cannot change the SQL standard. The user can nest another query and reduce the table to one column.